templates/troubleshooting.html

```diff
@@ -16,28 +16,28 @@ <h1 class="u-no-margin--bottom">Troubleshooting</h1>
   <div class="row">
     <div class="col-8 col-medium-4">
   <h2 id="troubleshooting-networking-issues">Troubleshooting networking issues</h2>
```

Comment:
Is the indentation supposed to look like this? Everything that's been edited seems to be indented to the same number of spaces, are you seeing the same?

Reply:
Done